vitest.config.ts

```diff
@@ -2,8 +2,12 @@ import { defineConfig } from "vitest/config";
 
 export default defineConfig({
   test: {
+    exclude: [
+      "**/node_modules/**",
+      "./test/.tmp/**"
```

Comment:
Added `temp` directory to exclutions to get rid of `skipped` tests in the the coverage report